### PR TITLE
Create How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_con…

### PR DESCRIPTION
**Pull request by tikr7**
@matzew as you requested I wrote a small blog post how we use Knative Eventing =)

### CODE REVIEW

**Comment by matzew** [comment]
@tikr7 thanks a lot!

Can you please add your blog article to the navigation? 

Done in here -> https://github.com/knative/docs/blob/main/blog/config/nav.yml#L51

File: blog/docs/articles/How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_construction_industry.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## How Conxai uses Knative Eventing to provide AI APIs in the construction industry
+
```
Comment (matzew) [inline blog/docs/articles/How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_construction_industry.md:2]: I'd add an author, like

**Author: NAME, TITLE @ SHOP**

Comment (tikr7) [reply blog/docs/articles/How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_construction_industry.md:4]: Done.

**Comment by tikr7** [comment]
> @tikr7 thanks a lot!
> 
> Can you please add your blog article to the navigation?
> 
> Done in here -> https://github.com/knative/docs/blob/main/blog/config/nav.yml#L51

Done.

If something is wrong, pls do let me know.

File: blog/docs/articles/How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_construction_industry.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Another Knative Trigger moves it into our transformer which handles preprocessing first. As the next step, the transformer calls the predictor (Nvidia Triton) directly over HTTP. We use the "Collocate transformer and predictor in same pod" [feature](https://kserve.github.io/website/0.13/modelserving/v1beta1/transformer/collocation/) to maximize inference speed and throughput. After receiving model results, the transformer does the postprocessing and finally saves the model results on S3.
```
Comment (matzew) [inline blog/docs/articles/How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_construction_industry.md:17]: Could we have a little sub-header above this blog as well? 

Saying something about KServe / AI ?

File: blog/docs/articles/How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_construction_industry.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Our blurring product step by step
+
+First, the customer uploads an image to be blurred via our API, which saves the image on S3. The S3 event is routed to our AWS EventBridge. The TriggerMesh AWSEventBridgeSource automatically configures an Amazon SQS queue and EventBridge Rule to forward the S3 event into that Amazon SQS queue. Because TriggerMesh has been deprecated, we will switch to Knative Serving [IntegrationSource](https://knative.dev/blog/articles/consuming_sqs_data_with_integrationsource/) approach with AWS SQS in the future. Then it passes that event into the Knative Broker which is backed by Amazon MSK - Kafka. Since we heavily rely on S3 key/file names, we wrote our own router service in Golang which modifies the EventType using regex patterns and sends it back to the Knative Broker.
```
[the remark this answers is not in the document]
Comment (tikr7) [reply blog/docs/articles/How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_construction_industry.md]: @pierDipi nice little detail, thank you! I changed "Serving" to "Eventing" :)

**Comment by tikr7** [comment]
> Thanks for sharing your adoption story @tikr7 !
> 
> In the future, I would love to know more about your router server to see if Knative could provide a generalized version of it and, at the same time, remove that from your plate.
> 
> A very small comment/suggestion

We can hop on a call at some point. Let's sync on Slack. I texted you a message.

File: blog/docs/articles/How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_construction_industry.md (hunk 2 above)
Comment (tikr7) [reply blog/docs/articles/How_Conxai_uses_Knative_Eventing_to_provide_AI_APIs_in_the_construction_industry.md]: Thank you for the feedback. I integrated something about Kserve.

**Review by matzew — APPROVED** [review]
/lgtm

**Comment by dprotaso** [comment]
/approve

If anyone has further edits feel free to do it in a follow up.

**Pull request merged**